### PR TITLE
biggie chews (tech bunker edit)

### PR DESCRIPTION
**Pull request by panzerr1944**
## About The Pull Request
i replaced the bad, easy to cheese var-edited lil' chew-chew in the tipton tech bunker with the big chew-chew ported from old DR that was meant to be in it's place (but never got put in for one reason or another, which is wild to me given that bunker spawns 2-shot weapons). i also slightly bumped up it's damage thanks to the results of testing.

on myrios' request with his help testing, we also edited the riot sentries out in favour of funny incin sentries (be scared), added a T-45d spawner and a superhigh cargo ballistic spawner and further fixed up big chew to be a true bossmob with his own resistance category. no ez meister 2 grenade cheese methods for you. more like 90 grenades now. or more.

further edited on request of Zero%Angel with their advice to make their armour good. because the armour code is spaghetti. resistances are close to that of a PA claw, so prepare for trouble, although it's luckily not double.

tl;dr- bunker gets more challenge. bossmob is an actual bossmob now. loot improved a bit to compensate.

if this gets merged, there will be an actual challenge to big chew and he will be an actual bossmob. people will die. it will be funny to watch.

also here's this image again:
image

## Pre-Merge Checklist
- [Y] You tested this on a local server.
- [Y] This code did not runtime during testing.
- [Y] You documented all of your changes.

## Changelog
add: added the real big-chew to the tech bunker
del: removed old lil' chew-chew var edit from tech bunker
balance: changed chew-chew beam dam. from 18 to 20
add: added a superhigh cargo ballistic spawner at the tech bunkers deathclaws
add: added a t-45d pa set to the end of the tech bunker
balance: gave big chew his own armour setup to make him a true bossmob (1 aer9 could reliably kill him before without too much issue, which was bad)
balance: chew attacks you while your uncon now, like a bossmob would.
balance: chew fires faster too.
del: riot sentries in tech bunker
add: incendiary slug sentries in the riot sentries place

### CODE REVIEW

File: code/__DEFINES/armor.dm [hunk 1]
```diff
@@ -369,6 +369,24 @@
...
+		"bullet" = 75, \
+		"laser" = 60, \
+		"energy" = 10, \
+		"bomb" = 80, \
+		"bio" = 100, \
+		"rad" = 100, \
+		"fire" = 0, \
+		"acid" = 0, \
+		"wound" = 0, \
+		"damage_threshold" = 15)
```
Comment (Zero-Percent-Angel) [inline code/__DEFINES/armor.dm:388]: hmmm I know this is probably not just a thing on this mob, but I'm not sure how I feel about mob DT, makes things like shot guns loading bucks completely useless on some mobs, I'd much rather you just set the resists higher give me a shout if there is a percentage damage resist you have in mind and I'll tell you the armour value.

Comment (panzerr1944) [reply code/__DEFINES/armor.dm:388]: That's cool. I just set it to the amount equal to a PA claw. I'll bump it down to buckshot levels after a bit of checking, and slightly increase the overall resistance to compensate
